dataland-internal-storage/src/main/resources/application-test.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
```

Comment:
Warum sind hier credentials hardcoded? Machen wir das sonst nicht immer über Umgebungsvariablen?

Reply:
Das sind die Einstellungen für die In-Memory DB für die Unit/Integration Tests